VideoWeb/VideoWeb.EventHub/Handlers/Core/EventHandlerBase.cs

```diff
@@ -158,6 +159,12 @@ await HubContext.Clients.Group(Hub.EventHub.VhOfficersGroupName)
                 .NewConferenceAddedMessage(conferenceId);
         }
 
+        protected async Task PublishAllocationHearingsMessage(string csoUserName, IList<HearingDetailRequest> hearings)
```

Comment:
This can go to the `AllocationHearingsEventHandler` because this method is only ever used by the `AllocationHearingsEventHandler` and not by other event handler

Reply:
why? All the other handlers have their methods in the event base

Reply:
Those should not be there. That's not a pattern, that's a bad understanding of inheritance.

Reply:
I have refactored all the method in the event base moving to the appropriate handler